test/test-files/volar/src/App.vue

```diff
@@ -0,0 +1,8 @@
+<script setup>
```

Comment:
The contents of this file are not needed to run the tests so this can be empty.  No need to to make ALE bigger than what it is already.